datalad_gooey/dataladcmd_exec.py

```diff
@@ -80,6 +80,9 @@ def _cmdexec_thread(self, cmdname, **kwargs):
         # for example.
         if 'result_xfm' not in kwargs:
             kwargs['result_xfm'] = None
+
+        dataset = dlapi.Dataset(kwargs.pop('dataset'))
+        kwargs['dataset'] = dataset
```

Comment:
This probably does not work, if not dataset argument was provided, which is a valid use case. I think it needs to be made conditional.